bitarray/sparse_bitarray.go

```diff
@@ -296,6 +300,67 @@ func (sba *sparseBitArray) IntersectsBetween(other BitArray, start, stop uint64)
 	return true
 }
 
+// WriteSparse serializes the sparseBitArray and passes it to the writer.
+func WriteSparse(w io.Writer, ba *sparseBitArray) error {
```

Comment:
This exported function has an unexported type in the argument list. This unusable from an external library. I'm unsure of how to solve this. I don't think it's ideal to export `sparseBitArray`.

@dustinhiatt-wf @tylertreat-wf thoughts?